config/samples/dataplane_v1beta1_openstackdataplaneservice.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  labels:
+    app.kubernetes.io/name: openstackdataplaneservice
+    app.kubernetes.io/instance: openstackdataplaneservice-sample
+    app.kubernetes.io/part-of: dataplane-operator
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: dataplane-operator
+  name: openstackdataplaneservice-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
why not use `configure-network` spec here?

Reply:
this one *is* the sample, so I didn't make it specific to configure network.